### DISCUSSION TITLE
全球开源项目办公室崛起：GOTC 深圳站 OSPO SIG 圆桌带来了什么？ - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#69] [Announcements]
# 全球开源项目办公室崛起：GOTC 深圳站 OSPO SIG 圆桌带来了什么？ - 「开源之道」

https://opensourceway.community/posts/open-source-program-office/2021-gotc-shenzheng-rise-of-ospo/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

类似和 FLOSS/自由|开源软件 相关的概念/术语越来越多, 

> OSPO

如果说是企业开始尝试正规化 开源 关系的组织部门,
其开设是个重要标志,

不过, 每年值得回访统计一下:

- 去年设立 OSPO 的企业, 今年部门还在嘛?
- 如果取消了为什么?
- 如果还在, 一年里关键成绩是什么?

可能会看到更多信息, 合规 这在传统企业一直是很清晰的行为, 吻合在地政策/规则,
但是, 对于 FLOSS/自由|开源软件 

- 合什么规? 
- 如何算是合规? 
- 合规成本有多少?
- 合规后潜在收益有什么? 有多少?
- ...

如果对应经济帐不清楚, 那么 OSPO 们其实很尴尬的...
